src/components/LeaderBoard.js

```diff
@@ -163,139 +53,120 @@ function LeaderBoard(props) {
-                textAlign="center"
-              >
-                Trees Planted ${country.planted}
-              </Box>
-            </Box>
-            <Box
-              classes={classes.countryText}
-              fontSize="20px"
-              fontWeight="200"
-              textAlign="center"
-              bottom="170px"
-              position="relative"
-              left="20px"
-              paddingRight="313px"
-              fontFamily="Montserrat"
+            <Grid
+              container
+              columns={16}
+              justifyContent="center"
+              alignItems="center"
             >
-              Country
-            </Box>
-            <Box
-              classes={classes.countryText}
-              fontSize="30px"
-              fontWeight="700"
-              textAlign="center"
-              bottom="170px"
-              position="relative"
-              left="20px"
-              paddingRight="313px"
-              fontFamily="Montserrat"
-            >
-              {country.name}
-            </Box>
+              <Grid item xs={1}>
+                {index === 0 ? (
+                  <EmojiEventsTwoToneIcon
+                    style={{ color: 'gold', marginLeft: '5' }}
+                  />
+                ) : (
+                  <p></p>
+                )}
+                {index === 1 ? (
+                  <EmojiEventsTwoToneIcon
+                    style={{ color: 'silver', marginLeft: '5' }}
+                  />
+                ) : (
+                  <p></p>
+                )}
+                {index === 2 ? (
+                  <EmojiEventsTwoToneIcon
+                    style={{ color: 'brown', marginLeft: '5' }}
+                  />
+                ) : (
+                  <p></p>
+                )}
+              </Grid>
+
+              <Grid item xs={1}>
+                <Typography variant="h5" className={classes.customText}>
+                  {index + 1}
+                </Typography>
+              </Grid>
+              <Grid item xs={4}>
+                <Avatar
+                  src={country.flag || countries[index].flag}
+                  variant="rounded"
+                  className={`${classes.flag}  ${classes.sizeAvatar}  `}
+                  borderRadius
+                />
+              </Grid>
+              <Grid item xs={5}>
+                <Typography variant="h5" className={classes.customText}>
+                  {country.name}
+                </Typography>
+              </Grid>
+              <Grid item xs={5}>
+                <Typography variant="h5" className={classes.customText2}>
+                  {country.planted.toLocaleString()}
+                  <ParkTwoToneIcon
+                    style={{ color: 'green', marginLeft: '5' }}
+                  />
+                </Typography>
+              </Grid>
+            </Grid>
           </Box>
-        ))}
-      <br></br>
-    </Grid>
+        ))
+      ) : (
+        <p>Siuuu</p>
```

Comment:
Is this for error handling? If so it should have a more appropriate message so that the purpose is clear to other developers. If this is only here to allow the use of the ternary operator then it should be switched back to the [neater form of short circuiting.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND)

Reply:
This is not the latest version of my code, that part of the code doesn't exist anymore, I haven't pushed my changes to the remote because I don't wanna fill the pull request with commits, but if you want me to push the changes to see them closely I'll do it.

Reply:
The problem with the ribbons is that I copied as an SVG directly from Figma and if I move them they tend to disappear, or I don't know if it is because of my inexperience.

Reply:
Could you look at my last commit @QuaidBartolomei? I think I need some help, I'm kinda blocked trying to make this as 1:1 as possible and also trying to make it responsive :/

Reply:
> The problem with the ribbons is that I copied as an SVG directly from Figma and if I move them they tend to disappear, or I don't know if it is because of my inexperience.

Just give them a class and position them absolutely with a negative margin and proper size.